File: 프로그래머스/1/135808. 과일 장수/과일 장수.js
function solution(k, m, score) {
    score.sort((a, b) => -(a - b));
    console.log(score);
    let temp = [];
    return score.reduce((acc, e, i) => {
        temp.push(e)
        if(i % m === m - 1) {
            acc += Math.min(...temp) * m;
            temp = [];
        }
        return acc;
    }, 0);
    
}

File: 프로그래머스/1/135808. 과일 장수/과일 장수.cpp
#include <bits/stdc++.h>
using namespace std;

int solution(int k, int m, vector<int> score) {
    int res = 0;
    sort(score.begin(), score.end(), greater<int>());
    
    vector<int> temp;
    for(int i = 0; i < score.size(); i++) {
        temp.push_back(score[i]);
        if(i % m == m - 1) {
            res += temp[(int)(min_element(temp.begin(), temp.end()) - temp.begin())] * m;
            temp.clear();
        } 
    }
    return res;
}